cluster-provision/k8s/1.19/provision.sh

```diff
@@ -2,12 +2,22 @@
 
 set -ex
 
-function docker_pull_retry() {
+KUBEVIRTCI_SHARED_DIR=/var/lib/kubevirtci
+mkdir -p $KUBEVIRTCI_SHARED_DIR
+cat << EOF > $KUBEVIRTCI_SHARED_DIR/common.sh
```

Comment:
maybe it will be better that it will be a file in the folder which is copied or something like that ?
instead of creating it with HEREDOC

Reply:
Hmm, AFAIK we're not copying scripts to the VMs, we run the scritps over ssh. We do copy manifests.
I have tried running the common.sh script before provision.sh/nodes.sh (with added exports) but the variable wasn't defined. I assume that I cannot share the variables this way then.
I could instead copy this common.sh file in cluster-provision/gocli/cmd/run.go... But I find this method quite simple. Do you insist on copying the file to the VMs?

Reply:
Nope i don't insist, its great as you did.
Thanks